src/client.js

```diff
@@ -478,6 +480,8 @@ class Client {
         this._transport = connectionParameters._options.transport;
         this._traceAPICalls = connectionParameters._options.traceAPICalls;
         this._observers = [];
+        // Callback function for session expiration
```

Comment:
Not sure this comment is really usefull.

Reply:
done